plugin.go

```diff
@@ -273,11 +275,11 @@ func handlerCreateEndpoint(w http.ResponseWriter, r *http.Request) {
 	defer brMap.Unlock()
 
 	//Generate a unique vhost-user port name
-	vhost_port := fmt.Sprintf("v_%s", ip)
+	vhostPort := fmt.Sprintf("v_%s", ip)
 
 	//Generate VPP-dpdk vhost-user interface:
 	cmd := "vppctl"
-	args := []string{"create", "vhost", "socket", fmt.Sprintf("/tmp/%s", vhost_port), "server"}
+	args := []string{"create", "vhost", "socket", fmt.Sprintf("/tmp/%s", vhostPort), "server"}
```

Comment:
Why the two step process. Can we not create it in the final location right away? Maybe  I am missing something.

Reply:
I was seeing errors in vppctl that indicated it couldn't create a file at a path that doesn't already exist.